great_expectations/expectations/core/expect_column_to_exist.py

```diff
@@ -30,33 +30,85 @@
 
 
 class ExpectColumnToExist(BatchExpectation):
-    """Expect the specified column to exist.
+    """Checks for the existence of a specified column within a table.
 
     expect_column_to_exist is a \
     [Batch Expectation](https://docs.greatexpectations.io/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_batch_expectations).
 
+    BatchExpectations are one of the most common types of Expectation. They are evaluated for an entire Batch, and answer a semantic question about the Batch itself.
+
     Args:
         column (str): \
             The column name.
 
     Other Parameters:
-        column_index (int or None): \
+        column_index (int or None, optional): \
             If not None, checks the order of the columns. The expectation will fail if the \
             column is not in location column_index (zero-indexed).
-        result_format (str or None): \
+        result_format (str or None, optional): \
             Which output mode to use: BOOLEAN_ONLY, BASIC, COMPLETE, or SUMMARY. \
             For more detail, see [result_format](https://docs.greatexpectations.io/docs/reference/expectations/result_format).
-        catch_exceptions (boolean or None): \
+        catch_exceptions (boolean or None, optional): \
             If True, then catch exceptions and include them as part of the result object. \
```

Comment:
```suggestion
            When the value is True, exceptions are included with the result object. \
```
Update all similar instances for consistency.